kernelci/bisect.py

```diff
@@ -120,3 +129,67 @@ def get_recipients(kdir, commit, to=set(), cc=set()):
     cc = cc.difference(to)
 
     return {'to': list(to), 'cc': list(cc)}
+
+
+def _lore_atom_query(subject):
+    query = urllib.parse.urlencode({'x': 'A', 'q': subject})
+    url = 'https://lore.kernel.org/lkml/?' + query
```

Comment:
This should probably be https://lore.kernel.org/r//? - that'll include any list archived there, not just LKML. IIRC /all/ is a synonym for that.

Reply:
OK that worked with `/all/`.  I've added a parameter with this as the default value.